spec/models/schedule_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Schedule, type: :model do
+  describe 'validation' do
+    context '#from' do
+      it 'is not nil' do
+        subject.from = nil
+        subject.valid?
+        expect(subject.errors[:from]).to_not be_empty
```

Comment:
Custom matcher for these might be nice instead of explicitly typing this stuff every time. Maybe something like `have_errors_on(:field_name)`